qiskit/wrapper/_wrapper.py

```diff
@@ -237,7 +26,7 @@ def compile(circuits, backend,
 
     Args:
         circuits (QuantumCircuit or list[QuantumCircuit]): circuits to compile
-        backend (BaseBackend or str): a backend to compile for
+        backend (BaseBackend): a backend to compile for
```

Comment:
Need to fix this in the docstring of `execute` as well.

Reply:
done